src/gui/qgsattributeformeditorwidget.cpp

```diff
@@ -37,6 +37,10 @@ QgsAttributeFormEditorWidget::QgsAttributeFormEditorWidget( QgsEditorWidgetWrapp
   , mIsMixed( false )
   , mIsChanged( false )
 {
+  mConstraintResultLabel = new QLabel();
```

Comment:
This looks leaky - as far as i can tell, the parent is not set until `setMode( MultiEditMode )` is called (which may be never).